packages/modules/viewer-react/src/services/BaseInitializer.ts

```diff
@@ -260,5 +261,6 @@ export const getIModelAppOptions = (
         options?.i18nUrlTemplate ??
         (viewerHome && `${viewerHome}/locales/{{lng}}/{{ns}}.json`),
     }),
+    realityDataAccess: new RealityDataAccessClient(),
```

Comment:
Do we want to always initialize one for users and not allow them to configure it?  I'm a little concerned about them not being able to configure it.

Reply:
Typically we've been adding ways for users to override our defaults only after they request it. Im not opposed to exposing it. I defer to @kckst8

Reply:
@calebmshafer do you expect this to be a common request? Keeping in mind that one of the goals of the Viewer is to obscure large amounts of boilerplate and excess configurability